comp/otelcol/ddprofilingextension/def/go.mod

```diff
@@ -1,14 +1,13 @@
 module github.com/DataDog/datadog-agent/comp/otelcol/ddprofilingextension/def
 
-go 1.23.0
+go 1.24
```

Comment:
You need to replace every `go 1.24` with `go 1.24.0` in go.mod files to fix the `update_go` check